### PR TITLE
face, palm and crosstalk tests

### PR DESCRIPTION
**Pull request by heitorgo1**
all tests show data on their respectives datashboards

### CODE REVIEW

File: gestureTest/config.yaml [hunk 1]
```diff
@@ -5,7 +5,30 @@ description: test gestures
...
 services:
-  fistWatch:
+  gestureTest:
     engine: gesture
     type: palm
+
+dataTypes:
+  palmLocation:
+    x: integer
+    y: interger
```
Comment (eighteyes) [inline gestureTest/config.yaml]: `integer`

**Comment by eighteyes** [comment]
Are these tested? I'm trying to run the crosstalkConsumer / Producer and it's failing

File: crosstalkConsumerTest/app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// app code goes here
+// matrix.init()....
+//
+// have fun
+
+
+// It's working, but error occurs, Channel Closed
+matrix.on(function(payload){
+    if (typeof payload.value === 'number')
```
Comment (eighteyes) [inline crosstalkConsumerTest/app.js]: this is coming in as { type: 'app-message', payload: { randVal: 29 } }

**Review by eighteyes — CHANGES_REQUESTED** [review]
please test the crosstalk apps in conjunction before sending a PR. 

```
(crosstalkConsumerTest)err [(crosstalkConsumerTest)]⁊
specificValue not formatted correctly
 specificEmitter { type: 'app-crosstalkConsumerTest-message',
  payload: { randVal: 'h' } }
```

Is the error I get when trying to run both.

**Comment by eighteyes** [comment]
Also fix conflicts please/

**Comment by eighteyes** [comment]
@heitorgo1 what's going on here?

**Comment by heitorgo1** [comment]
Sorry @eighteyes , haven't been able to solve the conflicts yet, our matrix was unavailable because @di3goleite was using it in presentations for some events around here. Now he's back, and as soon as I get the matrix I'll solve this.

**Comment by heitorgo1** [comment]
@eighteyes both crosstalkConsumerTest and crosstalkProducerTest are running without errors now, but I'd like to show you something strange.
Sometimes the emitted message comes in the Consumer with some additional attributes, sometimes it doesn't, here's an example: 

`*******************PAYLOAD********************* { randVal: 76, type: 'app-message' }`
`*******************PAYLOAD********************* { type: 'app-message', payload: { randVal: 76 } }`

The same variable `randVal` sometimes comes wrapped in a `payload` object, and sometimes it doesn't.
I had to deal with these strange situations in the code for it to work as expected.

**Comment by eighteyes** [comment]
@heitorgo1 closing this PR. please fix conflicts, issues and resolve

**Pull request closed without merging**